app/src/test/java/io/apicurio/registry/ibmcompat/IBMCompatApiTest.java

```diff
@@ -89,6 +89,7 @@ public void testGetSchemas() throws Exception {
         // Get the list of artifacts via ibmcompat API
         List<SchemaListItem> schemas = given()
             .when()
+                .param("per_page", 200)
```

Comment:
I just got bitten by this :-)
So I know why it is set to 200 ... but I have a better fix. ;-)

Reply:
I definitely prefer your fix 🙂 thanks for sorting that out.

Reply:
I found it running with -Pinfinispan, where -Pstreams didn't fail for me ... go figure :-)